src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/Services/ISemrushCachingService.cs

```diff
@@ -0,0 +1,10 @@
+﻿
```

Comment:
Similar to the comment below on `ISemrushService`, think we might rename this to better reflect it's nature, which is a thin wrapper over Umbraco's generic caching service.  And we can move the generic parameter to just the method, to make it more reusable.

I.e.:

```
    public interface ICacheHelper
    {
        bool TryGetCachedItem<T>(string key, out T item) where T : new();

        ...
    }

    public class CacheHelper : ICacheHelper
    {
        private readonly IAppPolicyCache _cache;

        public CacheHelper(AppCaches appCaches)
        {
            _cache = appCaches.RuntimeCache;
        }

        public bool TryGetCachedItem<T>(string key, out T item) where T : new()
        {
            var serializedItem = _cache.GetCacheItem<string>(key);

            item = string.IsNullOrEmpty(serializedItem)
                ? new T()
                : JsonConvert.DeserializeObject<T>(serializedItem);

            return !string.IsNullOrEmpty(serializedItem);
        }
        
        ...
```
It also might be better to return null as the `item` rather than `new T()`, as that's perhaps more expected if the method returns false.  If you do that, you can remove the `new()` constraint I've added here, though probably need to add a `class` one.

Lastly, it's a bit inconsistent to get a strongly typed item back, but add a serialized string representation of that item.  Could we make that consistent by adding the strongly typed item and serializing it within the class?